docs/user/openstack/README.md

```diff
@@ -69,6 +69,25 @@ In addition, it covers the installation with the default CNI (OVNKubernetes), as
 
 ## OpenStack Requirements
 
+Before starting to install OpenShift, ensure following OpenStack services are
+available, as there are couple of them which we are relying on:
+
+- Keystone
+- Neutron
+- Nova
+  - with Metadata service enabled
+- Placement
+- Glance
+- Storage solution for the image registry:
+  - Swift
+  - Cinder with dynamically-provisioned volumes
+  - Manila for ReadWriteMany (RWX) which is required for HA
```

Comment:
As far I know, CIRO won't provision the image-registry in a manila volume on day 1, but it's possible to move it there as a day 2 operation. To be double-checked...

Reply:
Perhaps just mentioning Manila would be enough?